dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -112,9 +113,9 @@ public EntryResource(ToolDAO toolDAO, DockstoreWebserviceConfiguration configura
     @ApiOperation(nickname = "updateAliases", value = "Update the aliases linked to a entry in Dockstore.", authorizations = {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Aliases are alphanumerical (case-insensitive and may contain internal hyphens), given in a comma-delimited list.", response = Entry.class)
     public Entry updateAliases(@ApiParam(hidden = true) @Auth User user,
-        @ApiParam(value = "Entry to modify.", required = true) @PathParam("id") Long id,
-        @ApiParam(value = "Comma-delimited list of aliases.", required = true) @QueryParam("aliases") String aliases,
-        @ApiParam(value = "This is here to appease Swagger. It requires PUT methods to have a body, even if it is empty. Please leave it empty.") String emptyBody) {
+                               @ApiParam(value = "Entry to modify.", required = true) @PathParam("id") Long id,
```

Comment:
This is an odd amount of whitespace, rogue typo/`tab`?

Reply:
This must have been some automatic thing by my IDE, I didn't do this myself.

Reply:
Maybe re-import codestyle.xml?